obal/data/roles/diff_brew_package/tasks/main.yml

```diff
@@ -1,121 +1,15 @@
-    | cut -d' ' -f1"
-  register: latest_brewed_rhel5
-  changed_when: False
-  when: rhel5_branch_exists | bool
-  tags:
-    - always
-
-- name: 'Look up current version of package in brew for EL6'
-  shell: "brew list-tagged --quiet \
-    satellite-{{ satellite_version }}-rhel-6-candidate \
-    {{ inventory_hostname }} \
-    | sort --reverse --key 1 --version-sort \
-    | head -n1 \
-    | cut -d' ' -f1"
-  register: latest_brewed_rhel6
-  changed_when: False
-  when: rhel6_branch_exists | bool
-  tags:
-    - always
-
-- name: 'Look up current version of package in brew for EL7'
-  shell: "brew list-tagged --quiet \
-    satellite-{{ satellite_version }}-rhel-7-candidate \
-    {{ inventory_hostname }}
-    | sort --reverse --key 1 --version-sort
-    | head -n1
-    | cut -d' ' -f1"
-  register: latest_brewed_rhel7
-  changed_when: False
-  when: rhel7_branch_exists | bool
-  tags:
-    - always
-
-- name: 'Calculate EL5 RPM version in local specfile'
-  command: "rpmspec --query '{{ spec_file_path }}' \
-    --define 'dist .el5' \
-    --define 'foremandist .{{ foremandist }}' \
-    {{ scl_prefix_option_str }} \
-    --queryformat='%{name}-%{version}-%{release}\n' \
-    --srpm"
-  register: latest_local_rhel5
-  changed_when: False
-  when: rhel5_branch_exists | bool
-  tags:
-    - always
-
-- name: 'Calculate EL6 RPM version in local specfile'
-  command: "rpmspec --query '{{ spec_file_path }}' \
-    --define 'dist .el6sat' \
-    --define 'foremandist .{{ foremandist }}' \
-    {{ scl_prefix_option_str }} \
-    --queryformat='%{name}-%{version}-%{release}\n' \
-    --srpm"
-  register: latest_local_rhel6
-  changed_when: False
-  when: rhel6_branch_exists | bool
-  tags:
-    - always
-
-- name: 'Calculate EL7 RPM version in local specfile'
-  command: "rpmspec --query '{{ spec_file_path }}' \
-    --define 'dist .el7sat' \
-    --define 'foremandist .{{ foremandist }}' \
-    {{ scl_prefix_option_str }} \
-    --queryformat='%{name}-%{version}-%{release}\n' \
-    --srpm"
-  register: latest_local_rhel7
-  changed_when: False
-  when: rhel7_branch_exists | bool
-  tags:
-    - always
-
-- name: "Set latest for brew and local"
-  set_fact:
-    latest_brewed_rhel5: "{{ latest_brewed_rhel5.stdout | default('') }}"
-    latest_brewed_rhel6: "{{ latest_brewed_rhel6.stdout | default('') }}"
-    latest_brewed_rhel7: "{{ latest_brewed_rhel7.stdout | default('') }}"
-    latest_local_rhel5:  "{{ latest_local_rhel5.stdout | default('') }}"
-    latest_local_rhel6:  "{{ latest_local_rhel6.stdout | default('') }}"
-    latest_local_rhel7:  "{{ latest_local_rhel7.stdout | default('') }}"
-  tags:
-    - always
-
-- name: "Compared local and brew package versions"
-  set_fact:
-    rhel5_build_needed: "{{ latest_brewed_rhel5 != latest_local_rhel5 }}"
-    rhel6_build_needed: "{{ latest_brewed_rhel6 != latest_local_rhel6 }}"
-    rhel7_build_needed: "{{ latest_brewed_rhel7 != latest_local_rhel7 }}"
-  tags:
-    - always
+    always
```

Comment:
can you make this `- always` again? technically both work, but we use the explicit dashed form everywhere else